src/services/leafletMapDefaults.js

```diff
@@ -101,69 +102,8 @@ angular.module('ui-leaflet').factory('leafletMapDefaults', function ($q, leaflet
         setDefaults: function (userDefaults, scopeId) {
             var newDefaults = _getDefaults();
 
-            if (isDefined(userDefaults)) {
```

Comment:
This does seem way less complicated and I am not really sure the reasoning behind the way it was. @elesdoar any thoughts?